src/layouts/Footer.js

```diff
@@ -1,14 +1,135 @@
 import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Container from '@material-ui/core/Container';
+import Link from '@material-ui/core/Link';
+import { Paper, withStyles, Grid, TextField, Button, AppBar, Typography, Toolbar, CssBaseline} from '@material-ui/core';
```

Comment:
Possible to put all material-ui/core imports in the same line? Like this:

`import { Container, Link, Paper, withStyles, Grid, TextField, Button, AppBar, Typography, Toolbar, CssBaseline} from '@material-ui/core';`

Reply:
Fixed in the latest push